en/net/other-programming-languages/integration-with-aspose-total-components/convert-pdf-pages-to-images-and-recognize-barcodes/_index.md

```diff
@@ -101,88 +101,60 @@ For the purposes of this example, first convert a page in a PDF file into an ima
 
 {{% /alert %}}
 
-##### **Programming Samples**
-**C#**
-
-{{< highlight csharp >}}
-
- //Create a PdfConverter object
-
-PdfConverter converter = new PdfConverter();
+##### Programming Samples
 
-//Bind the input PDF file
+```csharp
```

Comment:
DO use private static method declaration to wrap code snippets.
https://wiki.saltov.dynabic.com/bin/view/1KievGroup/Aspose.PDF%20for%20.NET/Development/User%20documentation/